recipes-qt/qt5/qtremoteobjects_git.bb

```diff
@@ -25,6 +25,6 @@ PACKAGECONFIG[tools-only] = "CONFIG+=tools-only"
 
 EXTRA_QMAKEVARS_PRE += "${PACKAGECONFIG_CONFARGS}"
 
-SRCREV = "f64e34be9ac4b7e92c63e47235c04471a1d40c93"
+SRCREV = "29394e35db43acb20e7b4d5e978a733f5b4232a6"
```

Comment:
This doesn't seem correct

It's copy paste from:
qtquicktimeline_git.bb:SRCREV = "29394e35db43acb20e7b4d5e978a733f5b4232a6"

will fix that.

Reply:
you are right, correct hash is 18ef1cdce7bc4c93415f38f1c220ab697aa75908

Reply:
I've fixed it to sha of the tag itself 1d8e4d3645c437cb0a5e5cdb4862949190f89259 in master-next.